app/policies/action_item_policy.rb

```diff
@@ -34,4 +34,8 @@ def reopen?
   def user_is_creator?
     board ? board.creator?(user) : record.board.creator?(user)
   end
+
+  def user_is_member?
+    board ? board.member?(user) : record.board.member?(user)
```

Comment:
in this case we are checking if this membership exists?

```ruby
def user_is_member?
  board.memberships.where(user_id: user.id).exists?
end
```

something like this